requirements.txt

```diff
@@ -1,66 +1,40 @@
-atomicwrites==1.3.0
 attrs==19.3.0
-backcall==0.1.0
 beautifulsoup4==4.8.1
-certifi>=2019.11.28
 cftime==1.0.4.2
-chardet==3.0.4
+coverage==6.0
 Cython==0.29.14
 dask==2.7.0
-decorator==4.4.1
-docopt==0.6.2
-docutils==0.15.2
-entrypoints==0.3
-filelock==3.0.12
 future==0.18.0
-future-fstrings==1.2.0
-gitdb2==2.0.6
-graphtools==1.4.1
-idna==2.8
+GDAL==3.3.2
```

Comment:
I have already prepared the requirements.txt with >= instead of ==, now we can apply this change since now lisflood-utilities is fixed and also the date fix are applied here (were giving issues otherwise). Environment.yml will be updated with the final environment used after all the changes, as well as the docker file. 
If I'm not wrong, pcraster is not needed as a pip package (it is installed as a conda-forge package).